deploy.sh

```diff
@@ -1,38 +1,45 @@
 #!/bin/sh
 
 # Scripts to deploy application to staging server
-# @todo #45:30min Continue implementing deploy scripts. Find out user 
-#  and domain of the staging server and replace scp and ssh with correct data.
-#  Add scripts in cron (like the one created in #47). Verify the web 
-#  application is running (execute couple fo curl requests).
+# @todo #126:30min Finish implementing deploy scripts. Add scripts in cron 
+#  (like the one created in #47). Verify the web application is running - 
+#  execute couple of curl requests.
 
 echo "-- Run tests"
 pytest
 
 echo "-- Creating staging tag"
 git tag -a staging-$tag -m "Rultor deploy staging-$tag"
 
+STAGING_SERVER=$(jq -r '.credentials.server.staging.address' ../credentials.json)
+STAGING_USER=$(jq -r '.credentials.server.staging.username' ../credentials.json)
+STAGING_KEY=../staging.id_rsa
+PG_STAGING_USER=$(jq -r '.credentials.postgres.staging.username' ../credentials.json)
+PG_STAGING_PASS=$(jq -r '.credentials.postgres.staging.password' ../credentials.json)
+
 echo "-- Copy application code to staging server"
-scp -r . user@staging-server:/app
+scp -i $STAGING_KEY -r . $STAGING_USER@$STAGING_SERVER:/app
 
 # add scripts in cron (like the one created in #47)
 # verify the webapplication is running
-ssh user@staging-server << EOF
-  echo "-- Creating database user: timless_user"
-  sudo -u postgres psql -c "CREATE USER timeless_user WITH 
+ssh -i $STAGING_KEY $STAGING_USER@$STAGING_SERVER << EOF
+  echo "-- Creating database user: $PG_STAGING_USER"
+  sudo -u postgres psql -c "CREATE USER $PG_STAGING_USER WITH 
     SUPERUSER
     CREATEDB
     CREATEROLE
     INHERIT
     LOGIN
-    ENCRYPTED PASSWORD 'timeless_pwd';"
+    ENCRYPTED PASSWORD '$PG_STAGING_PASS';"
   echo "-- Creating database: timelessdb_dev"  
   sudo -u postgres psql -c "CREATE DATABASE timelessdb_dev;"
   echo "-- Creating database: timelessdb_test"  
   sudo -u postgres psql -c "CREATE DATABASE timelessdb_test;"
   echo "-- REPLACE: add scripts to cron"
   cd /app
```

Comment:
@bkuzmic We need also to update external libraries, right?

Reply:
@emilianodellacasa What do you mean by external libraries? Do you mean scripts, like backup or something else?

Reply:
@bkuzmic No, I mean flask libraries. I am just asking to add a `pip install -r requirements.txt` here